CONTRIBUTING.rst

```diff
@@ -0,0 +1,118 @@
+Contributing
```

Comment:
Can this file be CONTRIBUTING.md? Having a mix of .md and .rst is surely confusing. I'm aware that this is still the case in projectmesa/mesa.

Reply:
Sure, changed it from `.rst` to `.md`.